packages/time-picker/test/validation.test.js

```diff
@@ -254,20 +163,6 @@ describe('validation', () => {
       timePicker.value = '12:00';
       expect(timePicker.checkValidity()).to.be.true;
     });
-
-    it('should be valid when committing a non-empty value', () => {
```

Comment:
**note:** The new tests ensure that the component always validates _after_ the value property update, so it's now enough to just assert that the validation passes / fails when having / not having a value.